argo-cd-apps/base/eaas/toolchain-member-operator/toolchain-member-operator.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: toolchain-member-operator-eaas
```

Comment:
To avoid errors with `kustomize build` I gave this appset a unique name to avoid conflicts with `toolchain-member-operator` under the `../member/` dir.

```
> kustomize build argoc-cd-apps/overlays/development`
...
Error: accumulating resources: accumulation err='accumulating resources from '../../base/eaas': '/home/amisstea/src/infra-deployments/argo-cd-apps/base/eaas' must resolve to a file': recursed merging from path '/home/amisstea/src/infra-deployments/argo-cd-apps/base/eaas': may not add resource with an already registered id: ApplicationSet.v1alpha1.argoproj.io/toolchain-member-operator.argocd-staging
```

There may be a clever way to reorganize or reuse the appset with kustomize but I haven't found it yet. The problem is this appset has a different cluster generator than the existing `toolchain-member-operator` appset. Any suggestions would be appreciated.

Reply:
As discussed, the only way I found to reuse the same applicationSet for 2 type of clusters(was host and member in my case for backup), is to label the clusters with a new label and change the applicationSet to use this new label.

In the current case, we could label member and eaas clusters with a new label, e.g. `appstudio.redhat.com/toolchain-member`